e2e/cypress/integration/onboarding/cloud_sysadmin_onboarding_spec.js

```diff
@@ -0,0 +1,413 @@
+        cy.get('.Card__body.expanded .InviteMembersStep').should('be.visible');
+
+        // # Enter email addresses
+        cy.get('#MultiInput_InviteMembersStep__membersListInput input').should('be.visible').type('a@b.c,b@c.d,c@d.e,d@e.f,e@f.g,f@g.h,g@h.i,h@i.j,i@j.k,j@k.l,k@l.m,', {force: true});
+
+        // * Verify that the error message shows
+        cy.get('.InviteMembersStep__invitationResults').should('contain', 'Invitations are limited to 10 email addresses.');
+
+        // * Verify that Send button is disabled until only 10 emails remain
+        cy.findByTestId('InviteMembersStep__sendButton').should('be.disabled');
+
+        // # Remove the last email
+        cy.get('#MultiInput_InviteMembersStep__membersListInput input').should('be.visible').type('{backspace}');
+
+        // * Verify that Send button is now enabled
+        cy.findByTestId('InviteMembersStep__sendButton').should('not.be.disabled');
+    });
+
+    it('Cloud Onboarding - Copy invite link', () => {
+        cy.apiCreateTeam('team').then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+
+            // # Stub out clipboard
+            const clipboard = {link: '', wasCalled: false};
+            cy.window().then((win) => {
+                cy.stub(win.navigator.clipboard, 'writeText', (link) => {
+                    clipboard.wasCalled = true;
+                    clipboard.link = link;
+                });
+            });
+
+            // # Get invite link
+            const baseUrl = Cypress.config('baseUrl');
+            const inviteLink = `${baseUrl}/signup_user_complete/?id=${team.invite_id}`;
+
+            // * Verify initial state
+            cy.wrap(clipboard).its('link').should('eq', '');
+
+            // * Make sure channel view has loaded
+            cy.url().should('include', `/${team.name}/channels/town-square`);
+
+            // # Click Invite members to the team header
+            cy.get('button.NextStepsView__cardHeader:contains(Invite members to the team)').should('be.visible').click();
+
+            // * Check to make sure card is expanded
+            cy.get('.Card__body.expanded .InviteMembersStep').should('be.visible');
+
+            // * Verify correct invite link is displayed
+            cy.findByTestId('InviteMembersStep__shareLinkInput').should('be.visible').and('have.value', `${baseUrl}/signup_user_complete/?id=${team.invite_id}`);
+
+            // # Click Copy Link
+            cy.findByTestId('InviteMembersStep__shareLinkInputButton').should('be.visible').and('contain', 'Copy Link').click();
+
+            // * Verify that button reads Copied
+            cy.findByTestId('InviteMembersStep__shareLinkInputButton').should('be.visible').and('contain', 'Copied');
+
+            // * Verify if it's called with correct link value
+            cy.wrap(clipboard).its('wasCalled').should('eq', true);
+            cy.wrap(clipboard).its('link').should('eq', inviteLink);
+        });
+    });
+});
```

Comment:
@devinbinnie Do we have TM4J for each test case? If none, I can help create TM4J test cases. Please let me know, thanks

Reply:
@saturninoabril That depends on what TM4J is 😆

Reply:
Cases have been added to JIRA